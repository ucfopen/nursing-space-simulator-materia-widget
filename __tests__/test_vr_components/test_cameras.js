import React from "react";
import renderer from "react-test-renderer";
import { Entity } from "aframe-react";
import Aframe from "aframe";

import CameraFP from "../../src/js/components/assets/camera_fp";
import CameraTP from "../../src/js/components/assets/camera_tp";

describe("Camera tests", () => {
	it("renders CameraFP correctly", () => {
		const cameraFPTree = renderer
			.create(<CameraFP position={{ x: 0, y: 0, z: 0 }} active={true} />)
			.toJSON();

		expect(cameraFPTree).toMatchSnapshot();
	});

	it("renders CameraTP correctly", () => {
		const cameraTPTree = renderer
			.create(<CameraTP position={{ x: 0, y: 0, z: 0 }} active={true} />)
			.toJSON();

		expect(cameraTPTree).toMatchSnapshot();
	});
});
