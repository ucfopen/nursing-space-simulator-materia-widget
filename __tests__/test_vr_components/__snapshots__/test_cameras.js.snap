// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Camera tests renders CameraFP correctly 1`] = `<a-entity />`;

exports[`Camera tests renders CameraTP correctly 1`] = `<a-entity />`;
